src/locales/strings/enUS.json

```diff
@@ -500,6 +500,7 @@
   "string_first_ecash_wallet_name": "My eCash",
   "string_first_eos_wallet_name": "My EOS",
   "string_first_holesky_wallet_name": "My Holesky",
+  "string_first_hyperevm_wallet_name": "My Hyperliquid",
```

Comment:
This is different from en_US.ts. Is it meant to be "My Hyperliquid" or "My HyperEVM"? If we do nothing, one of these will just stomp the other the next time the strings update.